server/src/assets/game/driver-game.html

```diff
@@ -274,19 +294,37 @@ <h5 id="serverModalTitle" class="modal-title"></h5>
 	</form>
 
 	<form id="drivingForm">
-		<div id="chosenDestination" class="row pt-4">
+		<div class="row">
+			<label class="col-auto col-form-label-lg"><b>
+				<span lang="de">Fahrzeit:</span>
+				<span lang="en">Driving Time:</span>
+			</b></label>	
+			<label class="col-auto col-form-label-lg">
+			<span id="timerMinute">00</span>:<span id="timerSecond">00</span>
+			</label>
+		</div>
+
+		<div class="m-3"></div>
+
+		<div id="chosenDestination" class="row">
 			<label class="col-auto col-form-label-lg"><b>
 				<span lang="de">Ziel:</span>
 				<span lang="en">Destination:</span>
 			</b></label>
 			<svg class="flag" viewBox="0 0 100 100">
 				<use id="destination" href="#flagDef" class="flagThemeBlank"></use>
 			</svg>
-		</div> 
+			<label class="col-form-label-lg">
+				<span lang="en"><span class="easy">Please drive your train to the destination and then stop</span></span>
+				<span lang="de"><span class="easy">Bitte fahren Sie mit Ihrem Zug zum Zielort und halten Sie dann an</span></span>
```

Comment:
I would rephrase the german part a bit:    
"Bitte fahren Sie Ihren Zug zum Ziel und halten Sie kurz vor dem Ziel an"